src/frontend/Komponenter/Oppgavebenk/CustomSelect.tsx

```diff
@@ -8,6 +8,7 @@ interface Props<U extends string> {
     options: Record<U, string>;
     sortDesc?: boolean;
     skalSkjuleValgetAlle?: boolean;
+    size?: 'medium' | 'small';
```

Comment:
Hvis vi evt. vil støtte flere verdier i fremtiden 
```suggestion
size?: SelectProps['size'];
```

Reply:
Kunne velge mellom to imports. Valgte `@navikt/ds-react`